### ISSUE TITLE
EU citizens

### ISSUE THREAD

**Issue by pns2050** [woganmay/laravel-gdpr-blocker#2]
Not an expert on this but from my understanding, GDPR is still effective for EU citizens living outside EU.

**Comment by woganmay** [comment]
That is correct, which is why the library doesn't claim to block EU citizens - only EU IP ranges.
